cfilament/src/API.h

```diff
@@ -0,0 +1,162 @@
+/*
```

Comment:
The name of the file is a bit generic, FilamentTypes or something like that would be better